libs/utils/include/celix_array_list.h

```diff
@@ -484,6 +525,16 @@ celix_status_t celix_arrayList_addVersion(celix_array_list_t* list, const celix_
 CELIX_UTILS_EXPORT
 celix_status_t celix_arrayList_assignVersion(celix_array_list_t* list, celix_version_t* value);
 
+/**
+ * @brief add string pointer entry to the back of the array list.
+ * @note The string will *not* be copied.
+ *
+ * @param list The array list.
+ * @param value The string value to add to the array list.
+ * @return CELIX_SUCCESS if the value is added, CELIX_ENOMEM if the array list is out of memory.
+ */
+CELIX_UTILS_EXPORT celix_status_t celix_arrayList_addString(celix_array_list_t *list, const char* value);
```

Comment:
This has already been declared in this header file, and thus should be removed.
And the documentation is incorrect, since the string will be copied for explicit string-type array.